.github/CODEOWNERS

```diff
@@ -12,3 +12,6 @@
 
 # image-info tool
 /tools/**/*osbuild*image*info*  @thozza
+
+# Specify code ownership for files containing 'coreos' in the filename
+*coreos* @dustymabe @jlebon @ravanelli
```

Comment:
At some point, I noticed that we also need to explicitly list `@osbuild/osbuild-reviewers` for the group to be considered for specific paths. Since you are modifying the file, it may be beneficial to add it to all lines, in addition to specific individuals.

Reply:
I didn't add osbuild-reviewers to the coreos line.  I think there it makes sense to require at least one of them to approve.  If we need to merge something without their approval (like, idk, a codestyle or comment change in a coreos-related file), we can make an exception and bypass the rule.